src/main/pageobjects/selectCategory.page.ts

```diff
@@ -0,0 +1,120 @@
+const chaiExpect: any = require("chai").expect
+
+export default class SelectCategory
+{
+    //Load Page
+    public async open(path: string): Promise<any>
+    {
+        await browser.url(path)
+    }
+
+    //Maximize Window
+    public get maxWin()
+    {
+        return browser.maximizeWindow()
+    }
+
+    //Get Page Url Address And Title
+    public async getUrlAndTitle(): Promise<any>
+    {
+        console.log(await browser.getUrl())
+        console.log(await browser.getTitle())
+    }
+
+    //WebDriverIO Assertions
+    public async wdioAssertions(input1: string, input2: string): Promise<any>
+    {
+        await expect(browser).toHaveUrlContaining(input1)
+        expect(await browser).toHaveTitleContaining(input2)
+    }
+
+    //Chai Assertions
+    public async chaiURLAssertions(input: string, inputType: string): Promise<any>
+    {
+        chaiExpect(await browser.getUrl()).to.equal(input)
+        chaiExpect(await browser.getUrl()).to.be.a(inputType)
+        chaiExpect(await browser.getUrl()).to.have.lengthOf(input.length)
+    }
+
+    public async chaiTitleAssertions(input: string, inputType: string): Promise<any>
+    {
+        chaiExpect(await browser.getTitle()).to.equal(input)
+        chaiExpect(await browser.getTitle()).to.be.a(inputType)
+        chaiExpect(await browser.getTitle()).to.have.lengthOf(input.length)
+    }
```

Comment:
add assertions in the test cases only, pages should contain only locators and its actions like click, setValue